app/javascript/app/pages/home/user-cases-section/user-cases-section-component.jsx

```diff
@@ -0,0 +1,177 @@
+import React, { Component } from 'react';
+import Slider from 'react-slick';
+import { Icon } from 'cw-components';
+// Assets
+import userIcon from 'assets/icons/user.svg';
+import politicianImg from 'assets/home/government@2x.jpg';
+import researcherImg from 'assets/home/researcher@2x.png';
+import activistImg from 'assets/home/activist@2x.png';
+
+import styles from './user-cases-section-styles.scss';
+
+const pagingTitles = ['Government official', 'Researcher', 'Journalist'];
+
+const settings = {
+  dots: true,
+  infinite: true,
+  autoplay: false,
+  speed: 500,
+  slidesToShow: 1,
+  fade: true,
+  dotsClass: 'userCasesPaging',
+  customPaging: i => (
+    <div>
+      <Icon icon={userIcon} theme={{ icon: styles.userIcon }} />
+      <p>{pagingTitles[i]}</p>
+    </div>
+  )
+};
+
+const anchorsProps = {
+  rel: 'noopener norefer',
+  className: styles.link,
+  target: '_blank'
+};
+
+class UserCasesSectionComponent extends Component {
+  render() {
+    return (
+      <section className={styles.container}>
+        <div className={styles.sliderWrapper}>
+          <Slider
+            ref={slider => {
+              this.leftSlider = slider;
+            }}
+            slidesToShow={1}
+            autoplay={false}
+            beforeChange={(current, next) => this.rightSlider.slickGoTo(next)}
+            fade
+          >
+            <div>
+              <img src={politicianImg} alt="Political building" />
+            </div>
+            <div>
+              <img src={researcherImg} alt="Library" />
+            </div>
+            <div>
+              <img src={activistImg} alt="Demonstration" />
+            </div>
+          </Slider>
+          <Slider
+            ref={slider => {
+              this.rightSlider = slider;
+            }}
+            beforeChange={(current, next) => this.leftSlider.slickGoTo(next)}
+            {...settings}
+          >
+            <div className={styles.slideWrapper}>
+              <h3 className={styles.slideTitle}>
+                How can Climate Watch help you?
+              </h3>
+              <ol className={styles.slideList}>
+                <li>
```

Comment:
hmm, maybe we could extracts the texts to a table and loop through them, it got kind of crowded here. True also for the `li`s beneath